### ISSUE TITLE
Add support for ninja build system

### ISSUE THREAD

**Issue by Dor1s** [facebook/infer#1071]
Ninja is a widely used and efficient build system. Huge open source projects like LLVM and Chromium us it. Infer doesn't seem to support ninja as of now, therefore losing some decent amount of potential users.

**Comment by mbouaziz** [comment]
That would be great indeed!
Let us know how we can help you add support for it

**Comment by Dor1s** [comment]
@mbouaziz I'm not going to add it myself :)

**Comment by mbouaziz** [comment]
Us neither :(

**Comment by mbouaziz** [comment]
Looking at #1028, there may be an easy way using compilation databases, but it still requires a fix

**Comment by Dor1s** [comment]
Yeah, it looks like I got it running locally, will see if that succeeds. I assume that adding support for a new build system is pretty straightforward. Is it so?

**Comment by jvillard** [comment]
Looking at this again, I left a comment on #1028: https://github.com/facebook/infer/issues/1028#issuecomment-474362764

**Comment by michakfromparis** [comment]
+1

**Comment by marek22k** [comment]
Anti-Stale comment
